### DISCUSSION TITLE
Color of Top Bar of Windows Explorer

### DISCUSSION THREAD

**Discussion by vdn20** [valinet/ExplorerPatcher#2301] [Q&A]
When I open a windows explorer, the top part of the explorer window is white, compared to the rest of it being a darker color:

<img width="575" alt="image" src="https://github.com/valinet/ExplorerPatcher/assets/141947992/12629839-a307-4195-9a9f-07f0cfba56a4">

If I go into Settings --> Personalization --> Color, and switch to a different Accent color it changes color to what I expects:

<img width="574" alt="image" src="https://github.com/valinet/ExplorerPatcher/assets/141947992/0c830418-cd3c-4d76-b55f-44ae09c9fd26">

However, when I close that explorer, and open another one, it goes back to the first one where it's a light color.

Have anyone encounter this or know the fix for it?  It happened a few days ago, and I tried updating to the latest version a few minutes ago, and playing with the settings in ExplorerPatcher, but no luck on finding the problem.

Thanks

**Comment by pyrates999** [comment]
Please post windows version and build.  You can find this by running winver.
Please post explorer patcher version.

Please include what settings you changed in explorer patcher properties application.

**Comment by vdn20** [comment]
Windows version 22H2 Build 22621.2428
ExplorerPatcher version 22621.2361.58.4
<img width="312" alt="image" src="https://github.com/valinet/ExplorerPatcher/assets/141947992/b5935db6-d67c-4864-839b-2fda4266633e">
I didn't change anything when it happened from before, but I was changing settings the yesterday to see if it anything would change it back, didn't seem like anything I tried in this area affected it.

**Reply by pyrates999** [reply]
Do you have dark mode enabled?  Try disabling it.

**Reply by vdn20** [reply]
Yes, I have dark mode enabled in Windows.  When I go back to light mode, it shows the top bar clearly, but it's too bright and prefer dark mode.  Is there a way to temporarily disable explorer patcher to see if it's a Windows thing or not without uninstalling it?

**Reply by pyrates999** [reply]
It might be a bug in EP then when dark mode is enabled.

**Reply by vdn20** [reply]
Not sure if this bit will be helpful or not.  I uninstalled EP and restarted my system, and it still has the same behavior.  I tried looking for things online, and saw a lot about going back to the default Windows theme, but that didn't help.

So not sure if it's a Windows bug or an EP bug, but I guess I'll have to live with it for now and hope whatever caused it will fix itself.

Thanks

**Reply by pyrates999** [reply]
Ok, then it's not a bug with EP it looks like.
